mmv1/templates/terraform/constants/vertex_ai_feature_group_feature.go.erb

```diff
@@ -0,0 +1,9 @@
+if v, ok := d.GetOk("feature_group"); ok {
```

Comment:
Is constants correct here? I'm not sure that this will compile.

Reply:
Oh, this is used in `pre_create` and `pre_delete`. Let's copy it into both of those spots and delete this file.